### PR TITLE
Bump pgx to 0.3.1

### PR DESCRIPTION
**Pull request by JamesGuthrie**

### CODE REVIEW

**Comment by JamesGuthrie** [comment]
Looks like we're running into this: https://github.com/zombodb/pgx/issues/432

**Comment by Hoverbear** [comment]
Sorry. :( 0.3.1 is being cut: https://github.com/zombodb/pgx/pull/435

File: src/aggregates/gapfill_delta.rs [hunk 1]
```diff
@@ -27,7 +27,6 @@ unsafe extern "C" fn gapfill_delta_final(fcinfo: pg_sys::FunctionCallInfo) -> pg
 }
 
 #[derive(Serialize, Deserialize, PostgresType, Debug)]
-#[pgx(sql = false)]
```
Comment (cevian) [inline src/aggregates/gapfill_delta.rs:30]: why this change?

**Review by cevian — APPROVED** [review]
